schema.sql

```
@@ -171,3 +171,20 @@ CREATE TABLE IF NOT EXISTS message_requests (
+    id          TEXT PRIMARY KEY,
+    activity_id TEXT NOT NULL,
+    title       TEXT NOT NULL,
+    description TEXT,
+    file_key    TEXT NOT NULL,
+    uploaded_by TEXT,
+    created_at  TEXT NOT NULL DEFAULT (datetime('now')),
+    FOREIGN KEY (activity_id) REFERENCES activities(id) ON DELETE CASCADE,
+    FOREIGN KEY (uploaded_by) REFERENCES users(id) ON DELETE SET NULL
+);
+
+CREATE INDEX IF NOT EXISTS idx_materials_activity ON course_materials(activity_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)